pkgs/applications/science/electronics/bitscope/common.nix

```diff
@@ -0,0 +1,68 @@
+{
+  atk,
+  buildFHSUserEnv,
+  cairo,
+  dpkg,
+  fetchurl,
+  gdk_pixbuf,
+  glib,
```

Comment:
One more nitpick! These parameters are commonly written in nixpkgs with comma-first style:

```
{ atk
, buildFHSUserEnv
, cairo
...
```